tests/buffer/buffer_constructors_common.h

```diff
@@ -11,12 +11,60 @@
 
 #include "../common/common.h"
 
-namespace {
+namespace buffer_constructors_common {
 using namespace sycl_cts;
 
 template <typename T, int size, int dims>
 class BufferInteropNoEvent;
 
+template <typename T, int dims, typename allocT> class BufferCheck;
+
+template <int dims>
+bool check_range(cl::sycl::range<dims> r1, cl::sycl::range<dims> r2) {
```

Comment:
I have the feeling you can just use `r1 == r2` according to https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:byval-semantics